src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java

```diff
@@ -0,0 +1,856 @@
+import com.synopsys.integration.alert.common.persistence.accessor.JobAccessor;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationFieldModel;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationJobModel;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
+import com.synopsys.integration.alert.common.persistence.model.RegisteredDescriptorModel;
+import com.synopsys.integration.alert.common.persistence.util.ConfigurationFieldModelConverter;
+import com.synopsys.integration.alert.common.rest.FieldModelProcessor;
+import com.synopsys.integration.alert.common.rest.model.AlertPagedModel;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.common.rest.model.JobFieldModel;
+import com.synopsys.integration.alert.common.rest.model.JobFieldStatuses;
+import com.synopsys.integration.alert.common.rest.model.JobIdsValidationRequestModel;
+import com.synopsys.integration.alert.common.rest.model.JobPagedModel;
+import com.synopsys.integration.alert.common.rest.model.ValidationResponseModel;
+import com.synopsys.integration.alert.common.security.authorization.AuthorizationManager;
+import com.synopsys.integration.alert.component.certificates.web.PKIXErrorResponseFactory;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.rest.exception.IntegrationRestException;
+
+public class JobConfigActionsTest {
+    private AuthorizationManager authorizationManager;
+    private DescriptorAccessor descriptorAccessor;
+    private ConfigurationAccessor configurationAccessor;
+    private JobAccessor jobAccessor;
+    private FieldModelProcessor fieldModelProcessor;
+    private DescriptorProcessor descriptorProcessor;
+    private ConfigurationFieldModelConverter configurationFieldModelConverter;
+    private GlobalConfigExistsValidator globalConfigExistsValidator;
+    private PKIXErrorResponseFactory pkixErrorResponseFactory;
+    private DescriptorMap descriptorMap;
+
+    @BeforeEach
+    public void init() {
+        authorizationManager = Mockito.mock(AuthorizationManager.class);
+        descriptorAccessor = Mockito.mock(DescriptorAccessor.class);
+        configurationAccessor = Mockito.mock(ConfigurationAccessor.class);
+        jobAccessor = Mockito.mock(JobAccessor.class);
+        fieldModelProcessor = Mockito.mock(FieldModelProcessor.class);
+        descriptorProcessor = Mockito.mock(DescriptorProcessor.class);
+        configurationFieldModelConverter = Mockito.mock(ConfigurationFieldModelConverter.class);
+        globalConfigExistsValidator = Mockito.mock(GlobalConfigExistsValidator.class);
+        pkixErrorResponseFactory = Mockito.mock(PKIXErrorResponseFactory.class);
+        descriptorMap = Mockito.mock(DescriptorMap.class);
+
```

Comment:
It might be better to add a JobConfigAction member to this class then initialize the object here. Cut down on some duplicate code.